covid-19.md

```diff
@@ -16,6 +16,16 @@ Considering this is an event for minors, we will be taking decisions with an abu
 
 We are in contact with Reading University Students' Union who are continually reviewing the situation, and mitigating the risks associated with Coronavirus.
 
+The only things that will stop us are:
```

Comment:
Maybe drop the "only" here? While the list is fairly comprehensive, my pendant mind can find lots of things which aren't present on the list which would still stop us from holding the event.

Not sure if I'm being over-pedantic though.

I think it's because I keep reading "The only things that will stop us" almost like it's a challenge being issued. Maybe something like "The things would prevent the competition from running"?